README.md

```diff
@@ -14,4 +14,27 @@ Visit [sst-simulator.org](http://sst-simulator.org) to learn more about SST.
 
 See [Contributing](https://github.com/sstsimulator/sst-core/blob/devel/CONTRIBUTING.md) to learn how to contribute to SST.
 
+### Getting Started
+
+#### Prerequistes 
+
+##### Red Hat 7 Required Packages
+
+```sh
+yum install gcc gcc-c++ python python-devel make automake git libtool libtool-ltdl-devel
+```
+
+#### Building SST-Core from the repo 
+
+```sh
+git clone https://github.com/sstsimulator/sst-core.git
+cd sst-core 
+./autogen.sh
+mkdir build
+cd build
+# Don't disable MPI if you have one you want to use
+../configure --disable-mpi --prefix=$PWD/../install
```

Comment:
Why is disabling MPI in the default instructions - we should mostly want people to build with MPI right? This is the fastest current method for making the PDES parallel/performant?

Reply:
Centos7 lets you yum install (I only tested this in the docker image) one of the following 
- openmpi 1.10.7
- openmpi 3.1.3
- mvapich 2.3
- mvapich 2.0a
- mpich 3.0
- mpich 3.2

Which one do you suggest that we tell our users to install?

Reply:
We only test against openmpi 4.0.5 currently

Reply:
Exactly :), so if we want a minimal getting started install that only depends on what comes in the default repos it needs to not use MPI.  

@nmhamster, I don't think every example we show users how to build (this PR, or a Dockerfile) absolutely must come with the kitchen sink and  batteries included.  We want people to test the code out and you don't need MPI to do that.

Reply:
In my opinion we want at least some instructions that provide the fastest easiest time to get to hello world where by hello world I mean 
```
$ /path/to/sst/install/dir/bin/sst --help
```

Reply:
@calewis - agreed on a number of the TPLs for elements etc, but I'd say MPI is pretty important for SST. As above, its the main mechanism by which we deliver a significant value add (parallelism in the DES). I'm not convinced it is "kitchen sink and batteries".

Reply:
One possibly is to have two sub-sections - a build with MPI and a build without MPI (in that order because with MPI is I'm going argue preferred).

Reply:
We should trust our users enough to realize that `../configure --disable-mpi --prefix=$PWD/../install` is not the only way to build sst or the most performant or the one that necessarily suits the needs of experts.  I even put a comment that if they have an mpi they want to use they should remove that flag.  

But our hey here is how you get to hello world instructions can't include building openmpi 4.0.5 from source.

Reply:
Kokkos has this as the first thing they tell you to do to build Kokkos:
```
cmake $srcdir \
  -DCMAKE_CXX_COMPILER=$path_to_compiler \
  -DCMAKE_INSTALL_PREFIX=$path_to_install \
  -DKokkos_ENABLE_OPENMP=On \
  -DKokkos_ARCH_HSW=On \
  -DKokkos_ENABLE_HWLOC=On \
  -DKokkos_HWLOC_DIR=$path_to_hwloc
  ```
  Is OPENMP the best choice 🤷‍♀️, it is the one most likely to work if you just want to get started.

Reply:
I would not claim this is the most reduced version of the equivalent even for Kokkos. OpenMP is the most sensible parallel programming model to get started, but enable Haswell and HWLOC are not required for a basic setup. The issue is users building successfully but having a poor performance experience - Kokkos has unfortunately hit this a few times and its a hassle because then users (who don't know what they are doing) write this in slides, papers and talks.

Reply:
We can pick one of those already available MPIs it just won't be tested.

Reply:
FWIW, for most simulations on-node MPI and threads provide almost identical performance.  However, I do agree that we want to push people toward MPI.  You could just change it the have the example leave out --disable-mpi and add a comment that if there is no MPI in your path add --disable-mpi.  It's a subtle difference, but does at least suggest that building with MPI is preferred.

Reply:
Okay but I want this to work if you copy paste it.

Reply:
@feldergast @nmhamster just pinging on how you would like to proceed.  We can either install an older mpi from yum or keep the `--disable-mpi` flag.  I want this code to be directly runnable so it's one of those options.

Reply:
Marking resolved since we can pass the tests with the system MPI